### PR TITLE
Add in-person language to CoC

### PR DESCRIPTION
**Pull request by rayegun**
@skygering @vchuravy

### CODE REVIEW

**Comment by skygering** [comment]
Do you think it is better to have a point person or to leave it vague and let people pick?

**Comment by rayegun** [comment]
Not sure. I think in-person that's difficult since then you have to find a single person. I think giving people options is probably better. But open to suggestions.

**Comment by vchuravy** [comment]
My plan was to have a dedicated phone number during the event and have the DEI sub-committee be the first point of contact and not a general email list. We could have a seperate email list, but potential reports have a high level of sensitivity.

**Comment by rayegun** [comment]
I was thinking a reports@juliacon.org or something. Which can either go just to DEI or DEI and executive. 

Is this a phone number we would set up specifically for this purpose? Like a Google voice?

File: 2024/coc.md [hunk 1]
```diff
@@ -19,13 +20,15 @@ Participants asked to stop any harassing behavior are expected to comply immedia
...
-We expect participants to follow these rules on all platforms used for conference interactions.
+We expect participants to follow these rules on all platforms used for conference interactions and at all conference venues.
 We think people should follow these rules outside event activities too!
 
 ## Reporting
 
-If you would like to report a CoC violation or discuss any incident that may have left you uncomfortable, please contact one of the Discord admins (usernames in red) or organizers (usernames in blue).
-They may involve other event staff to ensure your report is managed properly.
-You won't be asked to confront anyone and we won't tell anyone who you are without your permission.
+If you would like to report a CoC violation or discuss any incident that may have left you uncomfortable, please contact a committee member (listed [here](https://juliacon.org/2024/committee/)) in-person, a volunteer at the front desk, or email us at [juliacon@julialang.org](juliacon@julialang.org).
```
Comment (bvdmitri) [inline 2024/coc.md:28]:
```suggestion
If you would like to report a CoC violation or discuss any incident that may have left you uncomfortable, please contact a committee member (listed [here](https://juliacon.org/2024/committee/)) in-person, a volunteer at the front desk, or email us at \email.
```

**Comment by MaxxxvD** [comment]
@rayegun @skygering do we still want to merge this? It has been open since January

**Comment by skygering** [comment]
What do we think now? Are we still going to get a phone number as @vchuravy suggested?

I think regardless we can merge this PR and open another when/if we do get a phone number.

**Review by bvdmitri — APPROVED** [review]

**Comment by bvdmitri** [comment]
Thanks for working on this. I personally would abstain from putting your phone numbers on the web since you are risking getting tons of spam messages from third parties, but if you wish to just open a new PR.

**Pull request merged**